lib/src/checkers/checker_base.dart

```diff
@@ -52,29 +31,30 @@ abstract class CheckerBase extends ChangeNotifier {
     return paintBounds;
   }
 
-  bool shouldIgnore(Element? element) {
-    // Stupid hack to not highlight the inspector button as an accessibility
-    // issue
-    if (element != null) {
-      final inspectorButtonChain = [
-        'FloatingActionButton',
-        'Positioned',
-        'Stack',
-        'WidgetInspector',
-      ];
-
-      final chain = element
-          .debugGetDiagnosticChain()
-          .map((e) => e.toStringShort())
-          .skip(1)
-          .take(inspectorButtonChain.length)
-          .toList();
-
-      return listEquals(chain, inspectorButtonChain);
-    }
-
-    return false;
-  }
+  // TODO: Re-enable
```

Comment:
Is this necessary piece of code? It could be useful to provide more info why this is commented out or remove it from code for now.